dist/formats/specialist_document/frontend/schema.json

```diff
@@ -229,8 +229,7 @@
       "additionalProperties": false,
       "required": [
         "body",
-        "metadata",
-        "change_history"
```

Comment:
It is unintentional. Will add the relevant fix to GovukContentSchemas::FrontendSchemaGenerator